tests/test_wave_01.py

```diff
@@ -141,12 +146,13 @@ def test_moves_movie_from_watchlist_to_watched():
     # Assert
     assert len(updated_data["watchlist"]) is 1
     assert len(updated_data["watched"]) is 2
+    assert updated_data["watched"][1] == movie_to_watch
     
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
```

Comment:
This new assert works well! Consider making this test more robust by also checking that the first movie in the list is unchanged.